README.md

```diff
@@ -137,7 +137,7 @@ These derived images include a set of standard capabilities that enable many of
 - CMake 3.10.2
 - OpenJDK 7 (1.7.0_95), 8 (1.8.0_151), and 9 (9-internal)
 - Java Tools (Ant 1.9.6, Gradle 2.10, Maven 3.3.9)
-- Go 1.9.2
+- Go 1.9.3 and 1.8.6
```

Comment:
This is a nit, but switch the version ordering? (Like how the OpenJDK versions are specified).

Reply:
Agreed... done.